tests/eval_files/trio911.py

```diff
@@ -161,8 +161,8 @@ async def foo_while_5():
     while foo():
         yield  # error: 8, "yield", Statement("yield", lineno)
 
-        async def foo_nested_error():  # error: 8, "exit", Statement("yield", lineno+1)# error: 8, "exit", Statement("yield", lineno+1)
-            yield  # error: 12, "yield", Statement("function definition", lineno-1)# error: 12, "yield", Statement("function definition", lineno-1)
+        async def foo_nested_error():  # error: 8, "exit", Statement("yield", lineno+1)
+            yield  # error: 12, "yield", Statement("function definition", lineno-1)
```

Comment:
Think previous behaviour here was incorrect, and these lines should only give one error each.